llvm/lib/Target/AArch64/AArch64ISelLowering.h

```diff
@@ -477,11 +477,12 @@ enum NodeType : unsigned {
   MSRR,
 
   // Strict (exception-raising) floating point comparison
-  STRICT_FCMP = ISD::FIRST_TARGET_STRICTFP_OPCODE,
+  STRICT_FCMP,
```

Comment:
I'd probably suggest keeping the FIRST_STRICT/LAST_STRICT enum wrappers to assist anyone that adds additional opcodes in the future - same for other cases below.

Reply:
I restored those and replaced `switch`es with `if`s.